src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,26 @@
+import { Filter } from '../Filter';
+
+type Props = {
+  status: string,
+  setStatus: (status: string) => void,
+};
+
+export const Footer: React.FC<Props> = ({ status, setStatus }) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        4 items left
```

Comment:
you can get props of todo.  and take a correct length.

Reply:
Thanks done didn't cross my mind even